### ISSUE TITLE
Missing WaitNamedPipeA/WaitNamedPipeW

### ISSUE THREAD

**Issue by lundmark** [Leandros/WindowsHModular#23]
Also the adjacant error definition, ERROR_PIPE_BUSY is missing.

I think that these are the ones that needs adding:

```
#define ERROR_PIPE_BUSY 231L
BOOL WINAPI WaitNamedPipeA(LPCSTR lpNamedPipeName, DWORD nTimeOut);

```

**Comment by lundmark** [comment]
Will make some PR's :)

**Comment by lundmark** [comment]
Should be fixed in #25
